grow/pods/catalogs.py

```diff
@@ -135,9 +144,17 @@ def update(self, template_path=None, use_fuzzy_matching=False,
         template = pofile.read_po(template_file)
         super(Catalog, self).update(
             template, no_fuzzy_matching=(not use_fuzzy_matching))
-        self.save(ignore_obsolete=ignore_obsolete,
-                  include_previous=include_previous, width=width,
-                  include_header=include_header)
+        # Don't use gettext's obsolete functionality as it polutes files: merge
+        # into main translations if anything
+        if include_obsolete:
+            self.merge_obsolete()
+        self.obsolete = odict()
+        self.save(include_header=include_header)
+
+    def merge_obsolete(self):
+        """ Copy obsolete terms into the main catalog """
```

Comment:
Just a tiny style nit, but docstrings should end in punctuation and not be padded with space characters. ;) This isn't a blocker but I just wanted to raise this for consistency for future code you write!